### PR TITLE
bandcamp-dl 0.0.7-09 (new formula)

### PR DESCRIPTION
**Pull request by ibrokemypie**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/bandcamp-dl.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    url "https://files.pythonhosted.org/packages/b3/b2/238e2590826bfdd113244a40d9d3eb26918bd798fc187e2360a8367068db/six-1.10.0.tar.gz"
+    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    system "#{bin}/bandcamp-dl", "--version"
```
Comment (bfontaine) [inline Formula/bandcamp-dl.rb]: Please write a test that does something more substantial than printing the version. You could e.g. download a small album then assert the files exist.

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/bandcamp-dl.rb (hunk 1 above)
Comment (ibrokemypie) [inline Formula/bandcamp-dl.rb]: Have pushed new tests that make sure both artist+album and url parsing work correctly, and the files exist.

File: Formula/bandcamp-dl.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    sha256 "f71359a7e2de2f5ea1eceea7c1e3222f2560ee48e21eef6f96957bb5c2ebb94a"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    system "#{bin}/bandcamp-dl", "--artist=iamsleepless", "--album=rivulets"
+    system "[", "-e", "iamsleepless/rivulets/01\ -\ rivulets.mp3", "]"
```
Comment (bfontaine) [inline Formula/bandcamp-dl.rb]: Please use Ruby instead:
```ruby
assert File.exist?("iamsleepless/rivulets/01 - rivulets.mp3")
```

Comment (ibrokemypie) [inline Formula/bandcamp-dl.rb]: have done

**Review by bfontaine — APPROVED** [review]

**Comment by fxcoudert** [comment]
The version was incorrectly detected as `09` instead of `0.0.7-09` (which is, I have to say, a weird version scheme).

**Comment by ibrokemypie** [comment]
has been fixed

**Comment by fxcoudert** [comment]
Thanks @ibrokemypie for your contribution to Homebrew! We look forward to other pull requests from you in the future :)

**Pull request closed without merging**